.travis.yml

```diff
@@ -1,12 +1,22 @@
 language: java
-sudo: false
+sudo: required
```

Comment:
Um, not as far as I know.
![image](https://cloud.githubusercontent.com/assets/5948271/25071443/738f6d9a-22b7-11e7-97d5-eeef495b2e9b.png)